### PR TITLE
add schedule

### PR DESCRIPTION
**Pull request by basine**

### CODE REVIEW

**Comment by ekurutepe** [comment]
@basine please do not merge. I have some title updates from speakers in my email. I'll refresh everything and merge tomorrow.

File: _workshops/my-taxi-workshop.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+presentationDate: Monday, 27th May & Tuesday, 28th May (repeat)
+time: "14:15"
+duration: 2 h
+fee: free
+prerequisites: None
+location: MyTaxi Boat Stage, RadialSystem
+order: 3
+---
+
+**Sensor: How to develop apps with a state  of the art architecture**
```
Comment (basine) [inline _workshops/my-taxi-workshop.md]: state[  ]of  <- extra whitespace

File: _workshops/my-taxi-workshop.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+A good architecture like MVVM makes it easy to separate the view from the business logic.
+But how do you manage the state?
+Do your presenter, interactor or view model use unidirectional data flow?
+Do you have a single source of truth of your state?
+Is you code easy to test and refactor?
+What about Referential Transparency?
+
+We would like to show case how we jazzed up MVVM by adding a state machine and ideas from Redux to solve all our state related troubles._
```
Comment (basine) [inline _workshops/my-taxi-workshop.md]: troubles._ <- underscore

**Comment by ekurutepe** [comment]
@basine I think this one's ready to go

**Pull request merged**